tensorflow/lite/kernels/CMakeLists.txt

```diff
@@ -0,0 +1,262 @@
+#
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(googletest REQUIRED)
+find_package(nsync REQUIRED)
+find_package(re2 REQUIRED)
+
+# Generate the mutable schema_generated.h header for tests.
+set(SCHEMA_FILE ${TFLITE_SOURCE_DIR}/schema/schema.fbs)
+set(SCHEMA_GENERATED_ROOT ${CMAKE_CURRENT_BINARY_DIR}/schema)
+set(SCHEMA_GENERATED_PATH ${SCHEMA_GENERATED_ROOT}/tensorflow/lite/schema/mutable)
+set(SCHEMA_GENERATED_FILE ${SCHEMA_GENERATED_PATH}/schema_generated.h)
+
+# Use the util function in flatbuffer to generate the schema header.
+include(BuildFlatBuffers)
+set(FLATBUFFERS_FLATC_EXECUTABLE ${CMAKE_BINARY_DIR}/flatbuffers-flatc/bin/flatc)
+set(FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS
+  -c
+  --no-union-value-namespacing
```

Comment:
is this needed?

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/schema/BUILD#L108

Reply:
@terryheo 
Oh... I misunderstood the flatbuffer_cc_library() call.
https://github.com/tensorflow/tensorflow/blob/38fca54c85764aadb40f7999ce14aa04b29d8653/third_party/flatbuffers/build_defs.bzl#L15

The default option will be replaced by user.
I will remove the "no-union-value-namespacing" option in next commit.